packages/nuxt/src/index.ts

```diff
@@ -18,10 +18,15 @@ type Options<
 
 export function createEnv<
   TServer extends Record<string, ZodType> = NonNullable<unknown>,
-  TClient extends Record<string, ZodType> = NonNullable<unknown>,
+  TClient extends Record<
```

Comment:
I changed this to agree with the Next.js behavior (enforcing the prefix). Feel free to undo if there was a reason for this decision.